### PR TITLE
Sensei repo update

### PR DESCRIPTION
**Pull request by kwryankrattiger**
Update `url` and `git` attributes in Sensei package to point to new locations on [Github](https://github.com/SENSEI-insitu/SENSEI/)

### CODE REVIEW

**Comment by kwryankrattiger** [comment]
Tested using `spack fetch` for all semantic versions and `develop`

File: var/spack/repos/builtin/packages/sensei/package.py [hunk 1]
```diff
@@ -14,20 +14,20 @@ class Sensei(CMakePackage):
     Visit/Libsim, ADIOS, Python scripts, and so on."""
 
     homepage = "https://sensei-insitu.org"
-    url      = "https://gitlab.kitware.com/sensei/sensei/-/archive/v3.1.0/sensei-v3.1.0.tar.gz"
-    git      = "https://gitlab.kitware.com/sensei/sensei.git"
-    maintainers = ['sshudler']
+    url      = "https://github.com/SENSEI-insitu/SENSEI/releases/download/v3.2.1/SENSEI-3.2.1.tar.gz"
+    git      = "https://github.com/SENSEI-insitu/SENSEI.git"
+    maintainers = ['sshudler','rkrattiger']
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/sensei/package.py]:
```suggestion
    maintainers = ['sshudler', 'kwryankrattiger']
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Can you please double check if the list of maintainers is correct?

**Review by alalazo — APPROVED** [review]

File: var/spack/repos/builtin/packages/sensei/package.py (hunk 1 above)
Comment (kwryankrattiger) [reply var/spack/repos/builtin/packages/sensei/package.py]: Fixed this, sorry about that!

**Comment by alalazo** [comment]
Giving some time to @sshudler to review this. If it stays stale for more than a few days ping me and I'll merge.

**Comment by kwryankrattiger** [comment]
@alalazo Just checking in on this

**Pull request merged**